### PR TITLE
Adhering to the Contributor Covenant Code of Conduct

### PR DESCRIPTION
**Pull request by miguelff**
**Disclaimer: The following is a personal contribution, and does not necessarily reflect the views of GitHub, my employer.**

A code of conduct is important because it promotes moral behavior, acts as a guideline for ethical decision making, enhances the reputation of an organization, prevents negative legal effects, encourages positive relationships, acts as a reference for solving ethical dilemmas and prevents discrimination or harassment.

Harassment and abuse are significant factors in preventing the entire community of tech companies, user groups, open source projects, and other organizations and events from being truly diverse and welcoming to participation.

Also, a well-written code of conduct clarifies an organization's mission, values and principles, linking them with standards of professional conduct. 

This aligns well with this project's inferred mission of helping people becoming better programmers and software professionals.

@sdmg15 as this repository' maintainer, you are setting the example for thousands of other developers. You can contribute to making this community, GitHub, and the world a better place by fostering a welcoming, diverse and inclusive culture! You've got a great power in your hands, but with great power comes a great responsibility 🙂 

Thanks for creating this repository, which I personally think has helped (and could continue helping) a lot of people! :heart:

### CODE REVIEW

**Comment by demimismo** [comment]
This PR supersedes #125

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at sonkengmaldini@gmail.com. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (miguelff) [inline CODE_OF_CONDUCT.md]: @sdmg15 I'm not sure if this is your email address, please let me know if I should change it. Thank you.

Comment (sdmg15) [inline CODE_OF_CONDUCT.md]: Yeah that's my email :smiley:

**Comment by miguelff** [comment]
Thank you for merging this @sdmg15! 💞

**Pull request merged**